### PR TITLE
Screenshots with fastlane screengrab

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Is used to generate screenshots in 44 languages on three devices.
Can be merged later.

### CODE REVIEW

File: src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <!-- Allows for storing and retrieving screenshots -->
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+
+    <!-- Allows changing locales -->
+    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION"/>
+
+    <application
+        android:testOnly="false"/>
+</manifest>
```
Comment (mario) [inline src/debug/AndroidManifest.xml]: New line please.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.58.0"
+
+default_platform :android
+
+platform :android do
+  before_all do
+    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
```
Comment (mario) [inline fastlane/Fastfile]: Would love to remove all redundant/not used stuff.

**Comment by mario** [comment]
@tobiasKaminsky rebase and fix things I mentioned pls :)

**Comment by tobiasKaminsky** [comment]
As of wish by @jancborchardt (https://github.com/nextcloud/android/issues/1081#issuecomment-351358620) I will include mocked devices.

**Comment by tobiasKaminsky** [comment]
Here you can find them for phone & tablet: 
https://nextcloud.kaminsky.me/index.php/s/Hsp5M4Lq9yiy9zz

**Comment by tobiasKaminsky** [comment]
@mario @AndyScherzinger please review
And then, once we have enough translations, I will upload the screenshots to fdroid / gplay.

File: fastlane/Screengrabfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+use_tests_in_packages ['com.owncloud.android.screenshots']
+
+app_apk_path 'build/outputs/apk/generic/debug/android-generic-debug.apk'
+tests_apk_path 'build/outputs/apk/androidTest/generic/debug/android-generic-debug-androidTest.apk'
+
+# locales ['en-US']
+locales ['en-US', 'de-DE', 'en-GB', 'fr-FR', 'it-IT', 'pt-BR', 'pt-PT', 'sv-SE', 'sq-AL', 'sq-MK', 'iw-IL', 'ar-AR', 'bg-BG', 'da-DK', 'fi-FI', 'gl-ES', 'hu-HU', 'tr-TR', 'uk-UK', 'vi-VI', 'ro-RO', 'ru-RU', 'sr-SR', 'pl-PL', 'el-GR', 'ko-KR', 'nl-NL', 'ja-JP', 'no-NO', 'eu-ES', 'lt-LT', 'es-ES', 'es-419', 'zh-HK', 'zk-CN', 'is-IS', 'sr-SR', 'id-ID', 'cs-CZ', 'sl-SL', 'fa-FA' ]
+
+# clear all previously generated screenshots in your local output directory before creating new ones
+clear_previous_screenshots true
```
Comment (AndyScherzinger) [inline fastlane/Screengrabfile]: Can this file also get a new line?

**Comment by AndyScherzinger** [comment]
👍 fine by me :)

**Comment by tobiasKaminsky** [comment]
@mario ping :-)

**Comment by tobiasKaminsky** [comment]
@mario happy new year and please review :-)

**Comment by mario** [comment]
I will need a bribe for this xD

👍

**Comment by tobiasKaminsky** [comment]
Merging with only one approval as @AndyScherzinger is on vacation.

**Pull request merged**